### PR TITLE
Added Anime profile pic genrator to the current repository

### PR DESCRIPTION
**Pull request by AmolShelke2**
Hey, project admin as you assign me the issue that I will be adding my anime app to the current repository I'm creating the pull request regarding that I have added my project to the repository please go through my pull request and I hope that you will merge my pull request thank for assigning me the issue.

#fixes iSSUE [156](https://github.com/Dezenix/frontend-html-css-js/issues/156)

@Abirpal202049 kindly review this pull request.

### CODE REVIEW

File: Anime App/style.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  font-weight: normal;
+  background: linear-gradient(to right, #00afff, #00ff6c);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+  overflow: hidden;
+  margin: 0;
+}
```
Comment (Abirpal202049) [inline Anime App/style.css:18]: Target the body by id or class

File: Anime App/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Tech Stacks Used
+
+![HTML](https://img.shields.io/badge/html5%20-%23E34F26.svg?&style=for-the-badge&logo=html5&logoColor=white)
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![JS](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E)
+
+## owner of the Project
+
+[Amol Shelke❤️‍🔥](https://github.com/AmolShelke2)
```
Comment (Abirpal202049) [inline Anime App/README.md:13]: Change the readme according to the rest of the projects

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]

**Comment by AmolShelke2** [comment]
I will fixed it
Thanks

**Comment by AmolShelke2** [comment]
@Abirpal202049 I have done changes please give it a look

**Review by Abirpal202049 — APPROVED** [review]

**Comment by AmolShelke2** [comment]
Thank you so much @Abirpal202049🙂

**Pull request merged**